source/gameengine/VideoTexture/VideoFFmpeg.cpp

```diff
@@ -68,6 +68,7 @@ const double defFrameRate = 25.0;
 // constructor
 VideoFFmpeg::VideoFFmpeg(HRESULT *hRslt)
     : VideoBase(),
+      m_codec(nullptr),
```

Comment:
m_codec variable can be removed. It is declared, initialized and asigned but never used.